chroma-manager/tests/feature/cli/features/steps/standard_cli.py

```diff
@@ -1,8 +1,3 @@
-#
-# ========================================================
-# Copyright (c) 2012 Whamcloud, Inc.  All rights reserved.
-# ========================================================
-
```

Comment:
Does this need to be replaced with our new copyright header rather than just deleted?